sirepo/db_upgrade.py

```diff
@@ -96,6 +96,10 @@ def _20240524_add_role_user(qcall):
         """
     )
     qcall.auth_db.drop_table("user_role_invite_t")
+    qcall.auth_db.execute_sql(
+        f"INSERT INTO user_role_t (uid, role, expiration)"
+        + f"VALUES ((select uid from user_registration_t), {sirepo.auth_role.ROLE_USER}, NULL);"
```

Comment:
Does this work? I think ROLE_USER needs to be quoted:

`f"VALUES ((select uid from user_registration_t), '{sirepo.auth_role.ROLE_USER}', NULL);"`

Reply:
That is an old commit.